City-of-Helsinki/drupal-helfi-form-tool
# Feature/lom 176 profiilitietojen tuonti

Allows admin to select certain fields from Helsinki Profiili to be used in forms. Automatically imports selected fields.

Test form:
https://lomaketyokalu.dev.hel.ninja/en/form/todistusjaljennospyynto-tilaus

- Make new
- Go to some form admin, add profile webform component to form if not already, select wanted fields
- Go to frontend with authenticated user (non-admin) in incognito
- See that selected fields are prefilled from profile data
- Submit form
- See that you selected fields are added to form data

TODO: Authentication requirements, Error handling, ...?
